framework/libraries/README.md

```diff
@@ -8,38 +8,39 @@ To build/use any of these (using `new-library` as an example):
 
 2. Navigate to the top-level directory:
 
-	`$ cd new-library`
+    `$ cd new-library`
 
 3. Restore packages by typing the following:
 
-	`$ dotnet restore`
+    `$ dotnet restore`
 	
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
+    **Note:** Starting with .NET Core 2.0 SDK, you don't have to run [`dotnet restore`](https://docs.microsoft.com/dotnet/core/tools/dotnet-restore) because it's run implicitly by all commands that require a restore to occur, such as `dotnet new`, `dotnet build` and `dotnet run`. It's still a valid command in certain scenarios where doing an explicit restore makes sense, such as [continuous integration builds in Azure DevOps Services](https://docs.microsoft.com/azure/devops/build-release/apps/aspnet/build-aspnet-core) or in build systems that need to explicitly control the time at which the restore occurs.
 	
 		
 4. To build and package the library as a NuGet package, type the following:
 
-	`$ cd src/Library`
-	
-	`$ dotnet build`
-	
-	`$ dotnet pack`
-	
-	Check out the `/bin/Debug` directory to see the generated artifacts and `.nupkg`.
+    ```
+    $ cd src/Library
+    $ dotnet build
+    $ dotnet pack
+    ```
+
+    Check out the `/bin/Debug` directory to see the generated artifacts and `.nupkg`.
 
 5. To run unit tests (only applicable to `new-library`):
```

Comment:
@BillWagner, the instructions for running unit tests depends on the existence of `new-library`.

Reply:
@Youssef1313 That sounds right to me. We still have goals to have all our projects with unit tests. That's both time consuming, and difficult to justify for many of the small snippets. It's time consuming because of the scale of our samples. It's difficult because in the case of many small, focused snippets, adding test code would harm the clarity of the small samples.

The best course of action for this PR is to accurately reflect the current state of samples. We'll continue to discuss internally how best to raise the quality gates on our sample code.

Reply:
> The best course of action for this PR is to accurately reflect the current state of samples. We'll continue to discuss internally how best to raise the quality gates on our sample code.

Well, I didn't get what to do in this PR. Should I add the deleted snippets back ?

Reply:
> I didn't get what to do in this PR. Should I add the deleted snippets back ?

Don't add back the removed snippets. We will continue to remove code not used in the articles. Make any further modifications to the readme.md files so that they accurately reflect the current samples.

Reply:
@BillWagner, I've updated the readme.